domains/utsav.is-not-a.dev.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Porfolio website,
+
+  "domain": "is-not-a.dev",
+  "subdomain": "utsav",
+
+  "owner": {
+    "repo": "https://github.com/dr-hofstadter/dr-hofstadter.github.io",
+    "email": "utsav.raj.338@gmail.com"
+  },
+
+  "record": {
+    "A": ["185.199.108.153", "185.199.109.153"],
+    "AAAA": ["2606:50c0:8000::153", "2606:50c0:8001::153"],
+    "CNAME": "dr-hofstadter.github.io",
```

Comment:
You should note that if you are going to use CNAME records, you are not able to have other records.